tracing-subscriber/src/subscribe/mod.rs

```diff
@@ -686,7 +686,31 @@ pub(crate) mod tests;
 /// be composed together with other subscribers to build a [collector]. See the
 /// [module-level documentation](crate::subscribe) for details.
 ///
+/// # Enabling interest
+///
+/// Whenever an tracing event (or span) is emitted, it goes through a number of
+/// steps to determine how and how much it should be processed. The earlier an
+/// event is disabled, the less work has to be done to process the event, so
+/// subscribers should attempt to provide accurate information as early as
```

Comment:
maybe something like:
```suggestion
/// subscribers that implement filtering attempt to disable unwanted events as early as
```

Reply:
👍